Content.Server/Armor/ArmorSystem.cs

```diff
@@ -16,8 +19,49 @@ public override void Initialize()
 
             SubscribeLocalEvent<ArmorComponent, DamageModifyEvent>(OnDamageModify);
             SubscribeLocalEvent<ArmorComponent, GetVerbsEvent<ExamineVerb>>(OnArmorVerbExamine);
+            SubscribeLocalEvent<ArmorComponent, PriceCalculationEvent>(GetArmorPrice);
         }
 
+        private void GetArmorPrice(EntityUid uid, ArmorComponent component, ref PriceCalculationEvent args)
+        {
+            if (component.Modifiers == null)
+                return;
+
+            double price = 0;
+
+            double coefDefaultPrice = 2; // default price of 1% protection against any type of damage
+            double flatDefaultPrice = 10; //default price of 1 damage protection against a certain type of damage
+
+            var prototypeManager = IoCManager.Resolve<IPrototypeManager>();
```

Comment:
could just make this a dependency so that this doesn't resolve every time the price is checked